client.go

```diff
@@ -732,9 +734,36 @@ func (c *Client) logStderr(r io.Reader) {
 		line, err := bufR.ReadString('\n')
 		if line != "" {
 			c.config.Stderr.Write([]byte(line))
-
 			line = strings.TrimRightFunc(line, unicode.IsSpace)
-			c.logger.Named(filepath.Base(c.config.Cmd.Path)).Debug(line)
+
+			l := c.logger.Named(filepath.Base(c.config.Cmd.Path))
+			// If output is not JSON format, print directly as error
+			if !isJSON(line) {
```

Comment:
Awesome. This is actually even better because non-Go plugins (that don't have access to go-hclog) can just output structured JSON to still get structured logging on the host. So this is a net win for sure.